src/content/projects.json

```diff
@@ -29,7 +29,7 @@
       "date": "December 2019",
       "description": "2019 at Penn was a year to remember. This year saw Penn take center stage in national news, when former Penn men’s basketball coach Jerome Allen was ensnared in the college admissions scandal and when two former Penn professors launched their presidential campaigns. On campus, students challenged the administration on multiple fronts. Fossil Free Penn demanded climate action at a Board of Trustees meeting, students protested Penn Law professor Amy Wax’s controversial comments, and a petition postponed the renaming of Penn Law. Take a look back at this year’s most defining moments with The Daily Pennsylvanian’s Year in Review.",
       "sponsored": false,
-      "link": "https://dailypenn.github.io/old.projects.thedp.com/2019/12/year-in-review/",
+      "link": "/2019/12/year-in-review",
```

Comment:
it might be worth it to simply this path. @peterbaile why is the path /2019/12/year-in-review instead of like 2019/year-in-review